app/api/Torrent.js

```diff
@@ -48,8 +48,22 @@ export default class Torrent {
 
     this.engine.add(magnetURI, { path: cacheLocation }, torrent => {
       const server = torrent.createServer();
```

Comment:
This is a unnecessary thing I did. Lets remove this local variable and `this.server = torrent.createServer();`